Changelog.md

```diff
@@ -1,5 +1,8 @@
 ## Changes
 
+### 6.1.0
```

Comment:
I still think is a breaking change...but anyway is just a technicality x'D

Reply:
@mirko-plowtech don't worry, I'll fix it. A lot less painful here than in `all`, no jenkins to deal with :P